src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -436,7 +434,7 @@ def lookup_group_data_stacktrace_bulk(
                         CreateGroupingRecordData(
                             group_id=group_id,
                             project_id=project_id,
-                            message=messages[group_id],
+                            message=event.title,
```

Comment:
I'm not sure that `event.title` is right here, this is how we get the group title in ingestion https://github.com/getsentry/sentry/blob/251c09dde880f11d052a6120534edd3a7929c36b/src/sentry/event_manager.py#L1308

And event metadata comes from https://github.com/getsentry/sentry/blob/251c09dde880f11d052a6120534edd3a7929c36b/src/sentry/event_manager.py#L858

Reply:
Also, why we we using title in the message? Are you looking for the value from `Group.message`? https://github.com/getsentry/sentry/blob/4c13721d122e9b034b0dcfd99aea9c3a6c91d8e5/src/sentry/models/group.py#L531

Reply:
https://github.com/getsentry/sentry/blob/35206b51ef1e019122b98abf6c03043193058b95/src/sentry/grouping/ingest/seer.py#L149

this is how we're getting the title for events when we call seer in ingestion